java/ql/test/library-tests/dataflow/fluent-methods/Test.java

```diff
@@ -0,0 +1,52 @@
+package smowton;
```

Comment:
Does this compile, if the file is not in `smowton/Test.java`?

Reply:
I believe it compiles just fine (but may not be easily referenced). There is, however, no real need for putting this in a package, so we might as well remove this line.